backend/tests/recipe_test.py

```diff
@@ -177,3 +177,8 @@ def test_raises_when_last_step_has_neither_options_or_next(basic_recipe):
     basic_recipe["steps"][1]["options"] = []
     with pytest.raises(ValidationError, match='Step must have a valid value for either "next" or "options"'):
       MicrolabRecipe.model_validate(basic_recipe)
+
+
```

Comment:
I'm starting to think these are intentional

Reply:
i suggest to keep the pytest runner, as it helps me to quickly run tests locally and debug.
but feel free to remove them.